### PR TITLE
1002 - Fided unwanted code snippet (handled warning)

### PR DESCRIPTION
**Pull request by buddy4kp**
### All Submissions:

* [ ] Have you followed the [Pull Requests Guidelines](https://github.com/buddyboss/buddyboss-platform/wiki/Submitting-Pull-Requests#pull-request-guidelines)?
* [ ] Does your code follow the [WordPress' Coding Standards](https://make.wordpress.org/core/handbook/best-practices/coding-standards/)?
* [ ] Have you checked to ensure there aren't other open [Pull Requests](../pulls) for the same update/change?

### Changes proposed in this Pull Request:

Fixes https://github.com/buddyboss/buddyboss-platform/issues/1002

### How to test the changes in this Pull Request:

1. When my website visitors visit my website and register as a user, an error message appears above the user form itself. Have tried reinstalling the entire site but it does not help.

### Other information:

* [ ] Have you added an explanation of what your changes do and why you'd like us to include them?
* [ ] Have you written new tests for your changes, as applicable?
* [ ] Have you successfully run tests with your changes locally?

### Changelog entry

> Enter a summary of all changes on this Pull Request. This will appear in the changelog if accepted.

### CODE REVIEW

File: src/bp-core/bp-core-catchuri.php [hunk 1]
```diff
@@ -1185,7 +1185,7 @@ function bp_private_network_template_redirect() {
 					// Check if strict match
 					if ( false !== $check_is_full_url && $request_url === $un_trailing_slash_it_url ) {
 						return;
-					} elseif ( false === $check_is_full_url && isset( $request_url ) && isset( $un_trailing_slash_it_url ) && strpos( $request_url, $un_trailing_slash_it_url ) !== false ) {
+					} elseif ( false === $check_is_full_url && isset( $request_url ) && !empty( $request_url ) && isset( $un_trailing_slash_it_url ) && !empty( $un_trailing_slash_it_url ) && strpos( $request_url, $un_trailing_slash_it_url ) !== false ) {
```
Comment (mehulkaklotar) [inline src/bp-core/bp-core-catchuri.php:1188]:
```suggestion
					} elseif ( false === $check_is_full_url && !empty( $request_url ) && !empty( $un_trailing_slash_it_url ) && strpos( $request_url, $un_trailing_slash_it_url ) !== false ) {
```

File: src/bp-core/bp-core-catchuri.php [hunk 2]
```diff
@@ -1185,7 +1185,7 @@ function bp_private_network_template_redirect() {
 					// Check if strict match
 					if ( false !== $check_is_full_url && $request_url === $un_trailing_slash_it_url ) {
```
Comment (mehulkaklotar) [inline src/bp-core/bp-core-catchuri.php:1186]:
```suggestion
					if ( false !== $check_is_full_url && ( !empty( $request_url ) && !empty( $un_trailing_slash_it_url ) && $request_url === $un_trailing_slash_it_url ) ) {
```

Comment (mehulkaklotar) [reply src/bp-core/bp-core-catchuri.php:1186]: @buddy4kp here too we can check.

**Comment by buddy4kp** [comment]
@mehulkaklotar  I have applied changes as per suggestion and also checked locally. It looks good Now. Thanks.

**Pull request merged**